src/app/shared/constants/validation-messages.ts

```
@@ -0,0 +1,151 @@
+    },
+    mustContainLetters: {
+      base: this.MUST_CONTAIN_LETTERS
+    },
+    minNumberValue: {
+      base: this.LESS_THAN_PARTICIPANTS
+    },
+    invalidEmailType: {
+      base: this.INVALID_EMAIL_TYPE
+    }
+  };
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)